docs/hopsworksai/aws/eks_ecr_integration.md

```diff
@@ -131,37 +130,21 @@ Navigate to your newly created role in [AWS management console](https://console.
             "Effect": "Allow",
             "Action": "eks:DescribeCluster",
             "Resource": "arn:aws:eks:*:*:cluster/*"
-        },
-        {
```

Comment:
why to remove the S3 permissions?

Reply:
Because I don't think we should rewrite all the permissions here. There is a big risk of forgetting to update the permissions here when we update them in other places in the documentation. In this part of the documentation, we should just add permissions for EKS to the profile and not change the permissions that are set and explained in another part of the documentation.